core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -69,6 +70,7 @@ export class BubbleCluster extends Chart {
     this._onBubbleClick = onBubbleClick;
     this._bubblePadding = bubblePadding;
     this._filter = filter;
+    this._label = false || label;
```

Comment:
Maybe an empty string would be more type-appropriate?

Reply:
The label is still work in progress, but it won't be just a string. It will also have a background with some color. In that case, is it better if I add an empty string instead of "false"?

Reply:
Rule is to always use the same type. If it's an object, not only a string, an empty {} will also be falsish.